### PR TITLE
Port EMBEDDING_LOOKUP to TFLM

### LINKED ISSUE #1871: Port EMBEDDING_LOOKUP to TFLM
@tensorflow/micro 

Port the EMBEDDING_LOOKUP kernel operator to TFLM.
Recreate TfLite tests for kernel operator (INT8, FLOAT32).

### PR DESCRIPTION
**Pull request by ddavis-2015**
@tensorflow/micro

Port the EMBEDDING_LOOKUP kernel operator to TFLM.
Recreate tests for kernel operator (INT8, FLOAT32). 
Add SymmetricScaleFromMinMax method to test_helpers.h file. 
Update Makefiles.
Update Bazel build script.

bug=fixes #1871

### CODE REVIEW

**Comment by advaitjain** [comment]
@emmett-google, would you be up for a first review of this PR?

**Comment by emmett-google** [comment]
Looks like a good port of the TfLite op  👍 Thanks for implementing this!

File: tensorflow/lite/micro/kernels/embedding_lookup_test.cc [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+                            tensors[kOutputTensorIndex].dims->data[i]);
+  }
+}
+
+}  // namespace
+}  // namespace testing
+}  // namespace tflite
+
+// TODO(ahentz): write more tests that exercise the details of the op, such as
+// lookup errors and variable input shapes.
```
Comment (advaitjain) [inline tensorflow/lite/micro/kernels/embedding_lookup_test.cc:137]: let's remove this TODO.

File: tensorflow/lite/micro/kernels/embedding_lookup.cc [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+    default:
+      MicroPrintf("EMBEDDING_LOOKUP only supports FLOAT32 and INT8, got %s.",
+                  TfLiteTypeGetName(output->type));
+      return kTfLiteError;
+  }
+}
+
+}  // namespace
+
+TfLiteRegistration_V1 Register_EMBEDDING_LOOKUP() {
```
Comment (advaitjain) [inline tensorflow/lite/micro/kernels/embedding_lookup.cc:209]: this will now need to be TFLMRegistration

**Review by advaitjain — CHANGES_REQUESTED** [review]
apoligies for the delayed review.

File: tensorflow/lite/micro/kernels/embedding_lookup_test.cc (hunk 1 above)
Comment (ddavis-2015) [reply tensorflow/lite/micro/kernels/embedding_lookup_test.cc]: done

File: tensorflow/lite/micro/kernels/embedding_lookup.cc (hunk 2 above)
Comment (ddavis-2015) [reply tensorflow/lite/micro/kernels/embedding_lookup.cc]: done

**Review by advaitjain — APPROVED** [review]

**Comment by advaitjain** [comment]
@mergifyio refresh

**Comment by ddavis-2015** [comment]
@Mergifyio refresh

**Pull request merged**